foundation-extensions/mobile-core/lifecycle/README.md

```diff
@@ -20,16 +20,25 @@ Sessions contain information about the app's current lifecycle, such as the devi
 {% endtab %}
 
 {% tab title="iOS (AEP 3.x)" %}
+
+Add the AEPLifecycle extension and it's dependency, the [Mobile Core](../../mobile-core) extension, to your project using Cocoapods.
```

Comment:
It should be "... its dependency, the Mobile Core..."

Reply:
Sorry missed that before. Should be good now.